Dockerfile.dev

```diff
@@ -16,6 +16,7 @@ WORKDIR /app
 # ... really speeds up building new containers
 COPY requirements.txt /app/
 RUN pip install -r requirements.txt
+RUN pip install tox
```

Comment:
I could have a `requirements-test.txt` with the hash, but it would have just tox. Do we absolutely need to have hash verification for tox, seeing that it will never be installed in production?